scrapers/Algolia_Adultime.yml

```diff
@@ -2,33 +2,51 @@ name: "Adultime"
 sceneByURL:
   - action: script
     url:
+      - accidentalgangbang.com/en/video/
+      - adamandevepictures.com/en/video/
       - adulttime.com/en/video/
+      - AdultTimePilots.com/en/video/
```

Comment:
```suggestion
      - adulttimepilots.com/en/video/
```

Reply:
- the stash app matching from Scene URL text field to installed scrapers is case-sensitive
- the TitleCase variants are returned by Algolia API

if you search via Algolia API, some of the domains in the returned URLs are TitleCase, so if you save the returned data from the API, and then later if you only had lowercase in this list then the scraper does not match and is not usable

Ideally, the matching of Scene URL text field to installed scraper sceneByUrl url list would be case-insensitive, and all the URLs here could just all be lowercase, but it isn't so that's why I've done what I've done here

Reply:
I could add lowercase, but the TitleCase domain variant should stay as long as either:

- Algolia API keeps sending the domain as TItleCase
- stashapp Scene URL to sceneByUrl matcher is made case-insensitive

Reply:
I've added the lowercase variant

Reply:
>if you search via Algolia API, some of the domains in the returned URLs are PascalCase, so if you save the returned data from the API, and then later if you only had lowercase in this list then the scraper does not match and is not usable

so the issue is when using the search algolia api for searches sometimes the domain is not all lower?
that is easily fixable with a small change in the Algolia.py code

Reply:
exactly... thanks for the updated `Algolia.py` script... the lowercase conversion works a treat

removed the PascalCase variant here